src/scss/quadrone/items.scss

```diff
@@ -233,10 +233,6 @@
         }
       }
     }
-
-    .pills:not(:has(.pill)) {
```

Comment:
There is currently a case where we have an empty pill list, and it counts as an element for flexbox purposes, causing double gaps. Is there a different way we can go about this and still be able to hide empty pills containers?